### DISCUSSION TITLE
Serial Pins for programming via optiboot

### DISCUSSION THREAD

**Discussion by mjschurman** [SpenceKonde/DxCore#309] [Library discussion]
I apologize for asking such a basic question but I wanted to confirm the SERIAL pins that are used when programming using the optiboot boot loader.  In looking at the data sheet the default pins for USART0 are PA0 and PA1 (p 162 of the data sheet).  I am assuming these are the pins that should be used when uploading via serial.

**Comment by ObviousInRetrospect** [comment]
When you burn the bootloader you can choose the pin set (menu option)

**Reply by ObviousInRetrospect** [reply]
Tools->Bootloader Serial Port->pick the one you want
Tools->Burn Bootloader

**Comment by SpenceKonde** [comment]
When an Optiboot board def is selected, two additional submenus appear, one to set the pins and USART port, and the other to set the entry conditions.

**Reply by mjschurman** [reply]
Thank you for the help!
